db/migrations/002_add_columns_to_attributes.rb

```diff
@@ -0,0 +1,17 @@
+Sequel.migration do
+  change do
+    alter_table(:attributes) do
+      add_column :type, String, null: false
+      add_column :hidden, "boolean", default: false
+      add_column :read_only, "boolean", default: false
+      add_column :unique, "boolean", default: false
+      add_column :index, "boolean", default: false
+      add_column :loader, String
+      add_column :restricted, "boolean", default: false
```

Comment:
I updated the columns per https://github.com/mountetna/magma/issues/99#issuecomment-621507560, but this includes a few more that aren't listed there. Should we keep `unique`, `index`, and `loader`?

Reply:
I think `loader` will shortly be deprecated anyway; there is really no way to run them using the existing methodology (as hooks in the upload process) after the magma/metis changes. We need #116 to develop an alternative, but I think we should just abandon it for the moment. `unique` and `index` are okay, although I'm wondering about column proliferation now...

Reply:
Ok, I made `loader` non-editable in 5b00d86fd623b6cebaa301829c4980a7a4290539.